### PR TITLE
new package - Tenable Nessus Agent

### PR DESCRIPTION
**Pull request by rogisf2000**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111965&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Tenable,Inc.NessusAgent(x64)
```
Comment (stephengillie) [inline manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).installer.yaml:4]:
```suggestion
PackageIdentifier: Tenable.NessusAgent
```

File: manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Tenable,Inc.NessusAgent(x64)
```
Comment (stephengillie) [inline manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Tenable.NessusAgent
```

File: manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Tenable,Inc.NessusAgent(x64)
```
Comment (stephengillie) [inline manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).yaml:4]:
```suggestion
PackageIdentifier: Tenable.NessusAgent
```

File: manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).installer.yaml:4]: Please update file name and path to match this change.

File: manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).locale.en-US.yaml (hunk 2 above)
Comment (stephengillie) [reply manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).locale.en-US.yaml:4]: Please update file name and path to match this change.

File: manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).yaml (hunk 3 above)
Comment (stephengillie) [reply manifests/t/Tenable,Inc/NessusAgent(x64)/10.4.1.20157/Tenable,Inc.NessusAgent(x64).yaml:4]: Please update file name and path to match this change.

**Comment by rogisf2000** [comment]
requested changes committed.

**Comment by stephengillie** [comment]
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\t\Tenable,Inc\NessusAgent(x64)\10.4.1.20157', expected 'manifests\t\Tenable\NessusAgent\10.4.1.20157'

**Pull request closed without merging**